### PR TITLE
Signup feature

### PR DESCRIPTION
**Pull request by tunv**

### CODE REVIEW

File: app/controllers/registrations_controller.rb [hunk 1]
```diff
@@ -0,0 +1,12 @@
+class RegistrationsController < Devise::RegistrationsController
+
+  private
+
+  def sign_up_params
+    params.require(:user).permit(:name, :email, :password, :password_confirmation)
```
Comment (leminhtuan2015) [inline app/controllers/registrations_controller.rb]: ( )

File: config/routes.rb [hunk 2]
```diff
@@ -1,5 +1,5 @@
 Rails.application.routes.draw do
-  devise_for :users
+  devise_for :users, :controllers =>{registrations: "registrations"}
```
Comment (leminhtuan2015) [inline config/routes.rb]: controllers: {registrations: "registrations"}

File: app/controllers/registrations_controller.rb [hunk 3]
```diff
@@ -0,0 +1,12 @@
...
+  end
+
+  def account_update_params
```
Comment (phuongbk) [inline app/controllers/registrations_controller.rb]: Do you need it?

File: app/controllers/registrations_controller.rb [hunk 4]
```diff
@@ -0,0 +1,8 @@
+class RegistrationsController < Devise::RegistrationsController
+
+  private
```
Comment (phuongbk) [inline app/controllers/registrations_controller.rb]: Cái này dùng protected.

File: app/views/devise/registrations/new.html.erb [hunk 5]
```diff
@@ -0,0 +1,36 @@
...
+
+      <div>
+        <%= f.label :email %><br />
+        <%= f.email_field :email, autofocus: true, class: "form-control" %>
+      </div>
+
+      <div>
+        <%= f.label :password %>
+        <% if @minimum_password_length %>
+          <em>(<%= @minimum_password_length %> characters minimum)</em>
```
Comment (yuyuvn) [inline app/views/devise/registrations/new.html.erb]: i18n

**Comment by leminhtuan2015** [comment]
LGTM thanks.

**Pull request merged**